### ISSUE TITLE
Convert color names

### ISSUE THREAD

**Issue by Remo** [spatie/color#23]
I'm looking for a library that converts all my colors into the same format and stumbled up on your library. IT looks like a neat start. However, I also need to be able to convert a string like `red` to something like `#FF0000`. Any reason not to support that?

**Comment by sebastiandedeyne** [comment]
I don't think this could be added without a breaking change because it would need a change in the interface to do it good.

Planning to do a rework of this package at some point, I'll keep in mind to add this then.

**Comment by ian-patel** [comment]
How about tag categories to `#000000` as **Dark** and `#ffffff` ad **Light** color. 
```
Hex::fromString('#000000')->isDark()
```

**Comment by kevnk** [comment]
> How about tag categories to `#000000` as **Dark** and `#ffffff` ad **Light** color.
> 
> ```
> Hex::fromString('#000000')->isDark()
> ```

Check out #44

**Comment by freekmurze** [comment]
This already works via the Named class:\n\n```php\n$red = Named::fromString('red');\necho $red->toHex(); // #ff0000\necho $red->toRgb(); // rgb(255,0,0)\n```
